lib/enter.js

```diff
@@ -16,7 +16,7 @@ module.exports = function (tree) {
     if (transitions) {
       enter.classed('transitioning-node incoming-node', true)
       tree._forceRedraw() // Force a redraw so we see the updates transitioning
-      enter.classed('transitioning-node incoming-node', false)
+      enter.classed('transitioning-node', false)
     }
```

Comment:
Now that you're no longer removing the class `incoming-node`, it's going to stick around forever, which doesn't make sense. 

![screen shot 2016-12-07 at 4 07 21 pm](https://cloud.githubusercontent.com/assets/203161/20986595/3f58c73c-bc97-11e6-8cd7-132ffc3575df.png)